easybuild/easyconfigs/s/SeisSol/SeisSol-201703-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'SCons'
+
+name = 'SeisSol'
+local_commit_id = 'fb97bef'
+version = '201703'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.seissol.org'
+description = """SeisSol is a software package for simulating wave propagation and dynamic rupture based on
+ the arbitrary high-order accurate derivative discontinuous Galerkin method (ADER-DG)."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+sources = [{
+    'git_config': {
+        'url': 'https://github.com/SeisSol',
+        'repo_name': 'SeisSol',
+        'commit': version,
```

Comment:
Should this be changed to `'commit': local_commit_id`?

Reply:
it's not really needed. this commit is tagged as version 201703, and https://github.com/SeisSol/SeisSol/commit/201703 points to this commit.

I agree it looks a bit strange to have a version as the commit, but at least we're sure were downloading a tagged version instead of a randomly chosen commit.

Reply:
My point is that a tag like `201703` can be moved. A commit ID can't be moved.

Reply:
ok, fair enough.